tests/test_decorators.py

```diff
@@ -82,6 +97,27 @@ def returns_already_encoded_testview():
         response.headers["Content-Encoding"] = "some_encoding"
         return response
 
+    @test_blueprint.route('/check_logged_in')
+    @check_logged_in
+    def returns_logged_in_response():
+        response = make_response(RESPONSE_OBJ_VAL)
+        return response
+
+    @test_blueprint.after_request
+    def refresh_expiring_jwts(response):
+        try:
+            exp_timestamp = get_jwt()["exp"]
+            now = datetime.now(timezone.utc)
+            target_timestamp = datetime.timestamp(now + timedelta(minutes=30))
+            if target_timestamp > exp_timestamp:
+                access_token = create_access_token(identity=get_jwt_identity())
+                response = make_response(response, 201)
+                set_access_cookies(response, access_token)
+            return response
```

Comment:
We should import the method from the actual implementation and use it here. feels very easy to forget to update this if we change the function.

Reply:
How very DRY 😆😆😆 +1

Reply:
Which method are we looking to import here?